.circleci/config.yml

```diff
@@ -17,6 +17,7 @@ executors:
       - image: mbgl/android-ndk-r21e:latest
     working_directory: ~/code
     environment:
+      MBX_CI_DOMAIN: o619qyc20d.execute-api.us-east-1.amazonaws.com
```

Comment:
not a secret so confirming this is 💯  👍  

`The value of this variable is not a secret and so it can be committed to GitHub repositories if that is appropriate for your situation.`